core/nginx/letsencrypt.py

```diff
@@ -39,8 +45,30 @@
 # Wait for nginx to start
 time.sleep(5)
 
+class MyRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        self.path = '/var/empty/'
+        return SimpleHTTPRequestHandler.do_GET(self)
+
+def serve_one_request():
+    with HTTPServer(("0.0.0.0", 8008), MyRequestHandler) as server:
+        server.handle_request()
+
 # Run certbot every day
 while True:
+    while True:
+        hostname = os.environ['HOSTNAMES'].split(' ')[0]
+        target = f'http://{hostname}/.well-known/acme-challenge/testing'
```

Comment:
Shouldn't we test with a known existing file served by this conatiner?

Reply:
That's cf090b42 ; 204 should be unique enough to avoid false positives.